src/Command/CreateClassCacheCommand.php

```diff
@@ -57,7 +57,10 @@ public function __construct(string $cacheDir, bool $debug)
 
     public function configure(): void
     {
-        $this->setDescription('Generate the classes.php files');
+        $this->setDescription('Generate the classes.php files')
+            ->setName(static::$defaultName)// BC for symfony/console < 3.4.0
+            // NEXT_MAJOR: drop this line after drop support symfony/console < 3.4.0
```

Comment:
2 things:
- you're targetting master, BC-breaks are authorized here. Also, you will not see your changes for a long time if you commit here, and sf < 2.8 support will be dropped before that happens.
- sf < 3.4 is no longer maintained, IMO we should drop support for it.

Reply:
Which branch I should target? Or maybe need to drop support from composer.json symfony version 3.4 < ?

Reply:
Regarding the base branch, please take a look at https://github.com/sonata-project/SonataAdminBundle/blob/3.x/CONTRIBUTING.md#the-base-branch.

Reply:
done